requirements.txt

```diff
@@ -1,4 +1,4 @@
-agogos>=0.2.0
+agogos==0.2.0
```

Comment:
Agogos should just be the latest version

Reply:
But I guess I'll just change it if I update the package.

Reply:
When you have both a `pyproject.toml` and `requirements.txt`, you want your `pyproject.toml` to be as flexible as possible in terms of what versions work, but you want your `requirements.txt` to act like a lock file, decribing what exact versions of packages work in case your `pyproject.toml` restrictions are too loose. So your `pyproject.toml` dependencies should work, but your `requirements.txt` dependencies work for sure